server/src/routes/api/v1/reviewRouter.js

```diff
@@ -0,0 +1,23 @@
+import express from "express"
+import { Review } from "../../../models/index.js"
+import { ValidationError } from "objection";
+import cleanUserInput from "../../../services/cleanUserInput.js";
+
+const reviewRouter = new express.Router({ mergeParams: true })
+
+reviewRouter.post("/", async (req,res)=>{
+    const newReview = cleanUserInput(req.body)
+    const {content , userId} = newReview
+    const activityId = req.params.id
```

Comment:
`req.user` will give you the currently signed in user that passport is managing. the current user should not be sent in the POST request. the incoming POST request should only be data provided by the user